win_build_vs.bat

```diff
@@ -28,6 +28,7 @@ set build_coap=OFF
 set build_jni=OFF
 set build_SQL=OFF
 set build_AWS=OFF
+set build_SFTP=OFF
```

Comment:
should we enable this in one of the windows jobs?

Reply:
Two reasons I didn't do:
1) Tests require Maven and Java, so they wouldn't run with current instance configs
2) Win builds sometimes already reach timeout, didn't want to add further stuff, especially longer tests like this. (can take minutes to start up the java sftp mock server)